backend/api/settings.py

```
@@ -218,30 +222,40 @@ class ThrottleScopes:
+                    },
+                )
+            except Exception as e:
+                self.warning(
+                    "Failed to execute Discord Webhook for error logging: %s\nIf you don't have Webhooks set up, leave the environment variable empty.",
+                    e,
+                )
 
     def db_error(self, msg, *args, **kwargs):
         self.error("Database error: %s", msg, *args, **kwargs)
 
+    def error(self, msg, *args, **kwargs):
```

Ruff (0.15.21)

[warning] Missing type annotation for `*args`

(ANN002)